sql/executor.go

```diff
@@ -72,6 +75,8 @@ func (e Executor) Execute(args driver.Request) (driver.Response, int, error) {
 			txn.SetSystemDBTrigger()
 		}
 		planMaker.txn = txn
+		now = time.Unix(0, 0).Add(time.Duration(planMaker.session.UnixTimestamp))
```

Comment:
`proto.Timestamp.GoTime()` does this differently. I'm not sure if it matters, but it would be good be consistent. For that matter, it might be easier if `session.UnixTimestamp` were a `proto.Timestamp` so that you could just use the `GoTime()` method.